### PR TITLE
Fixed toolschan/crash for QAM tests, poo#17660

### PR DESCRIPTION
**Pull request by mimi1vx**
Now in DISTRI=sle if isnt defined REPO_0 it expects released version SLE
and handle official debug repositories instead playing with openQA
repositories

### CODE REVIEW

**Comment by mimi1vx** [comment]
http://argus.suse.cz/tests/170

File: tests/toolchain/crash.pm [hunk 1]
```diff
@@ -19,40 +19,46 @@ sub install_kernel_debuginfo {
     # list installed kernels
     assert_script_run 'rpmquery -a kernel-default*';
     # kernel debug symbols are huge, this can take a while
-    assert_script_run 'zypper ref; zypper -n -v in kernel-default-debuginfo', 1200;
+    script_run 'zypper ref';
+    zypper_call('-v in kernel-default-debuginfo');
```
Comment (dzedro) [inline tests/toolchain/crash.pm]: 1200 timeout have been **pretty big**, hopefully 30 & 700 seconds will be enough

Comment (mimi1vx) [inline tests/toolchain/crash.pm]: is, Piglit expands to 2Gig and 700sec is enough:)

**Comment by okurz** [comment]
typo in commit message and please adhere to http://chris.beams.io/posts/git-commit/ for commit messages. Better phrase as "Fix toolchain/crash for QAM tests (poo#17660)"

**Pull request merged**